votes/0625-new-triage-method-for-cherrypick.md

```diff
@@ -0,0 +1,56 @@
+# New Triage Process for Cherrypick
+
+Authors: @ichn-hu @Mini256 @zhangyangyu
+
+## Motivation
+
+According to the [release model](https://pingcap.github.io/tidb-dev-guide/project-management/release-train-model.html), TiDB is going to have multiple releases. Each release matches one git branch. For `type/bug` issues with `severity/critical` and `severity/major`, it is anticipated to be fixed on any [currently maintained release versions](https://pingcap.github.io/tidb-dev-guide/project-management/release-train-model.html#what-is-tidb-version-mechanism-under-release-train-model) if affected. Contributors and reviewers are responsible to settle the affected versions once the bug is identified as `severity/critical` or `severity/major`. For versions that are identified affected, a corresponding `affects-x.y` label will be added to the issue. Then for the pull request solving the issue, contributors and reviewers should add corresponding `needs-cherry-pick-x.y` labels. ti-srebot will create cherry-pick pull requests to each branch according to these labels.
```

Comment:
I remember that only the critical bugs should be fixed on every affect version.

Reply:
The rule has already been changed to both critical and major. It's decided by the release manager and applied in the recent releases.